ship_cheap.rb

```diff
@@ -25,20 +25,44 @@
   EasyPost.api_key = ENV['EASYPOST_PROD_KEY']
 end
 
-shipping_config = YAML.load_file(opts[:file])
+class ShippingConfig
+  attr_reader :from, :to, :package, :customs_info, :customs_items
 
-from_address = EasyPost::Address.create(shipping_config['from'])
-to_address = EasyPost::Address.create(shipping_config['to'])
-parcel = EasyPost::Parcel.create(shipping_config['package'])
+  def initialize(file)
+    @config = YAML.load_file(file)
+    @from = @config['from']
+    @to = @config['to']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.